app/Jobs/RegisterUser.php

```diff
@@ -68,6 +68,7 @@ public function handle(): User
             'github_username' => $this->githubUsername,
             'confirmation_code' => Str::random(60),
             'type' => User::DEFAULT,
+            'bio' => '',
```

Comment:
I understand this is a bit pointless, but the browserkit test fails without this as it can't find a `bio` property on the user model.